rdt-app/src/main/java/io/ona/rdt_app/listener/RDTImageListener.java

```diff
@@ -0,0 +1,47 @@
+package io.ona.rdt_app.listener;
+
+import android.graphics.Bitmap;
+import android.widget.ImageView;
+import android.widget.RemoteViews;
+
+import com.android.volley.toolbox.ImageLoader;
+
+import org.smartregister.util.OpenSRPImageListener;
+
+import edu.washington.cs.ubicomplab.rdt_reader.callback.OnImageSavedCallBack;
+import io.ona.rdt_app.BuildConfig;
+import io.ona.rdt_app.application.RDTApplication;
+import io.ona.rdt_app.util.RDTJsonFormUtils;
+
+/**
+ * Created by Vincent Karuri on 02/07/2019
+ */
+public class RDTImageListener extends OpenSRPImageListener {
```

Comment:
This class is not being used anywhere

Reply:
It will be used to sync images by entity id.

I prefer to keep it for the future.